templates/.github/workflows/go-test.yml

```diff
@@ -10,13 +10,14 @@ jobs:
         go: [ "1.18.x", "1.19.x" ]
     env:
       COVERAGES: ""
-      SKIP32BIT: false
     runs-on: ${{ format('{0}-latest', matrix.os) }}
     name: ${{ matrix.os }} (go ${{ matrix.go }})
     steps:
       - uses: actions/checkout@v3
         with:
           submodules: recursive
+      - id: config
+        uses: protocol/.github/.github/actions/read-config@master
       - uses: actions/setup-go@v3
```

Comment:
Could we reduce duplication? Around here we'd set `env.SKIPPED=contains(fromJSON(steps.config.outputs.json).skipOSes, matrix.os)`. Then reuse SKIPPED below.